### PR TITLE
Update the doc of autoExec part.

### PR DESCRIPTION
**Pull request by wangfh666**
Fixed a note about initializing the chain in the automatic execution contract.

### CODE REVIEW

**Comment by wangfh666** [comment]
@kaikai1024   Please review

File: docs/cita/system/auto-exec.md [hunk 1]
```diff
@@ -25,7 +25,20 @@ f95b72de: contAddr()
 
 ## 操作示例
 
-*首先需要启动一条链，具体方法见快速入门部分*
```
Comment (kaikai1024) [inline docs/cita/system/auto-exec.md:28]: Not only this part.

please check the https://github.com/cryptape/citahub-docs/issues/46

**Review by kaikai1024 — CHANGES_REQUESTED** [review]

**Comment by kaikai1024** [comment]
And the title.

**Comment by kaikai1024** [comment]
@ouwenkg You can help to do it.

**Review by kaikai1024 — APPROVED** [review]

**Comment by boundless-forest** [comment]
Need to update docs in the following directory? @kaikai1024 
https://github.com/cryptape/citahub-docs/blob/master/website/translated_docs/zh-CN/version-0.20.2/cita/configuration/chain-configuration.md

https://github.com/cryptape/citahub-docs/blob/master/website/translated_docs/eh-Us/version-0.20.2/cita/configuration/chain-configuration.md

File: docs/cita/system/auto-exec.md [hunk 2]
```diff
@@ -25,7 +25,20 @@ f95b72de: contAddr()
...
+```bash
+$ ./env.sh ./scripts/create_cita_config.py create \
+    --super_admin "0x4b5ae4567ad5d9fb92bc9afd6a657e6fa13a2523" \
+    --contract_arguments SysConfig.autoExec=true \
+    --nodes "127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003"
+```
+
+其中 `autoExec` 为自动执行开关。
+
+启动链接下来的步骤见[快速搭链](../getting-started)部分。
```
Comment (kaikai1024) [inline docs/cita/system/auto-exec.md]: 快速搭链 -> 快速入门

**Review by kaikai1024 — CHANGES_REQUESTED** [review]

**Comment by kaikai1024** [comment]
> Need to update docs in the following directory? @kaikai1024
> https://github.com/cryptape/citahub-docs/blob/master/website/translated_docs/zh-CN/version-0.20.2/cita/configuration/chain-configuration.md

@Keith-CY

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**